ChangeLog

```diff
@@ -20,6 +20,9 @@
  * Removed deprecated Cobyla/TNC.setIgnoreFailure method
  * Removed deprecated OptimizationAlgorithm.setMaximumEvaluationNumber method
  * Removed deprecated OptimizationResult.getEvaluationNumber
+ * Deprecated BayesDistribution in favor of JointByConditioningDistribution
+ * Deprecated ConditionalDistribution in favor of DeconditionedDistribution
```

Comment:
If we decide to only use IntervalConditionedDistribution, this will have to change again I guess (not an objection, it is reasonable to do this, I just want to make sure).

Reply:
Yes it is the plan